### ISSUE TITLE
Commit id

### ISSUE THREAD

**Issue by Hetox123** [Solar-Tweaks/SolarTweaks-Mappings#3]
I am getting the error wrong lunar version 
And i have used the correct mappings but when i click on the downloaded mappings it does not give any response. Pls don't ask me to make a discord ticket .if you can reply here it'll be appreciated

**Comment by DizzyDagem4** [comment]
yeah same for be but this time there is no mapping for my commit Id (85a12cc) and the discord got term and there has been no link yet :/
